packages/components/src/components/Slider/__tests__/Slider.ct.tsx

```diff
@@ -0,0 +1,227 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { expect, type MountResult, test } from '@playwright/experimental-ct-react';
+import { type Locator, type Page } from '@playwright/test';
+import sinon from 'sinon';
+
+import { Slider } from '../Slider';
+
+const SLIDER_TEST_ID = 'test-slider';
+const ARIA_LABEL = 'test slider';
+
+async function dragSlider(
+    page: Page,
+    container: MountResult | Locator,
```

Comment:
As `MountResult` extends `Locator`

```suggestion
    container: Locator,
```

Reply:
oh nice!